pxc-80-backup/Dockerfile

```diff
@@ -77,14 +79,18 @@ RUN cp /usr/share/doc/percona-xtrabackup-80-*/LICENSE /licenses/LICENSE.xtraback
 
 RUN curl -o /usr/bin/kubectl \
         https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl \
+    && curl -o /usr/bin/vault.zip \
```

Comment:
I think we need to check checksum or gpg key of this file 
https://www.hashicorp.com/security

Reply:
@delgod has removed vault cli, so problem is gone